[TensorExpr] Kernel: make prim::ConstantChunk handled like other ops.

ZolotukhinM opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #65555
* #65859
* #65852
* #65843
* #65554
* #65553
* #65552
* #65551
* #65550
* __->__ #65549

Previously it had a special handling, with this change it follows the
same mechanism as other ops.

Differential Revision: [D31148924](https://our.internmc.facebook.com/intern/diff/D31148924)

Commit:
[TensorExpr] Kernel: make prim::ConstantChunk handled like other ops.

Previously it had a special handling, with this change it follows the
same mechanism as other ops.

[ghstack-poisoned]

ZolotukhinM commented:
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31148924).

navahgar commented on torch/csrc/jit/tensorexpr/kernel.cpp:
```diff
@@ -2450,29 +2463,21 @@ Tensor TensorExprKernel::computeValue(const torch::jit::Value* v) {
+  if (op == prim::ConstantChunk) {
+    auto const& n = v->node();
+    argInputs.push_back(toArg(inputs[0]));
+    argInputs.push_back((int64_t)v->offset());
```
"static_cast<int64_t>" instead of the traditional cast.

navahgar reviewed (approved):
LGTM

Commit:
rebase on "[TensorExpr] Kernel: make prim::ConstantChunk handled like other ops."


Previously it had a special handling, with this change it follows the
same mechanism as other ops.

Differential Revision: [D31148924](https://our.internmc.facebook.com/intern/diff/D31148924)

[ghstack-poisoned]

ZolotukhinM commented:
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31148924).

Commit:
Update on "[TensorExpr] Kernel: make prim::ConstantChunk handled like other ops."


Previously it had a special handling, with this change it follows the
same mechanism as other ops.

Differential Revision: [D31148924](https://our.internmc.facebook.com/intern/diff/D31148924)

[ghstack-poisoned]

ZolotukhinM commented:
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31148924).

ZolotukhinM commented:
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31148924).

Commit:
Update on "[TensorExpr] Kernel: make prim::ConstantChunk handled like other ops."


Previously it had a special handling, with this change it follows the
same mechanism as other ops.

Differential Revision: [D31148924](https://our.internmc.facebook.com/intern/diff/D31148924)

[ghstack-poisoned]

ZolotukhinM commented:
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31148924).

Commit:
Update on "[TensorExpr] Kernel: make prim::ConstantChunk handled like other ops."


Previously it had a special handling, with this change it follows the
same mechanism as other ops.

Differential Revision: [D31148924](https://our.internmc.facebook.com/intern/diff/D31148924)

[ghstack-poisoned]

ZolotukhinM commented:
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31148924).

Commit:
Update on "[TensorExpr] Kernel: make prim::ConstantChunk handled like other ops."


Previously it had a special handling, with this change it follows the
same mechanism as other ops.

Differential Revision: [D31148924](https://our.internmc.facebook.com/intern/diff/D31148924)

[ghstack-poisoned]

ZolotukhinM commented:
@ZolotukhinM has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31148924).
